bin/reth/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "reth"
```

Comment:
Should probably rename the bin in case people were to `cargo install`, wouldn't want to overwrite